### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by gup444555** [share121/inter-knot#984] [其他]
![image](https://github.com/user-attachments/assets/1721e032-8a7c-4d7c-bf90-7519e3cf11b3)

**Comment by GT025233** [comment]
这个委托我接了！

**Reply by GT025233** [reply]
安装暴力猴，安装步骤某站找一下就行，然后再进这个网页点安装小助手就行了

**Comment by mwk07** [comment]
tampermonkey直接搜这个，然后下载

**Comment by 1451374815** [comment]
那你是怎么发帖子的

**Comment by gup444555** [comment]
OK兄弟们解决了，换了个浏览器

**Comment by Yizakl** [comment]
帖子GitHub也能发哦
